apps/auth0/tsconfig.json

```diff
@@ -5,6 +5,11 @@
     "rootDir": "src",
     "outDir": "build"
   },
+  "ts-node": {
```

Comment:
@peterstarling  should put this in the BE config to avoid doing the export before ts node calls

Reply:
do we need it though? in same places we had it in-line i was able to drop it (or should drop) because we changed the type to module

Reply:
Better one place than repeating it in multiple places